server/apps/publish/publish_filters/filter_condition_tests.py

```diff
@@ -394,3 +414,23 @@ def test_if_fc_is_used(self):
         with self.app.app_context():
             self.assertTrue(f._get_referenced_filter_conditions(1).count() == 1)
             self.assertTrue(f._get_referenced_filter_conditions(2).count() == 0)
+
+    def test_check_similar(self):
+        f = superdesk.get_resource_service('filter_conditions')
+        filter_condition1 = {'field': 'urgency', 'operator': 'in', 'value': '2'}
+        filter_condition2 = {'field': 'urgency', 'operator': 'in', 'value': '3'}
+        filter_condition3 = {'field': 'urgency', 'operator': 'in', 'value': '1'}
+        filter_condition4 = {'field': 'urgency', 'operator': 'in', 'value': '5'}
+        filter_condition5 = {'field': 'urgency', 'operator': 'nin', 'value': '5'}
+        filter_condition6 = {'field': 'headline', 'operator': 'like', 'value': 'tor'}
+        with self.app.app_context():
+            cmd = VocabulariesPopulateCommand()
+            filename = os.path.join(os.path.abspath(
+                os.path.dirname("apps/prepopulate/data_initialization/vocabularies.json")), "vocabularies.json")
+            cmd.run(filename)
+            self.assertTrue(len(f._check_similar(filter_condition1)) == 2)
+            self.assertTrue(len(f._check_similar(filter_condition2)) == 1)
+            self.assertTrue(len(f._check_similar(filter_condition3)) == 0)
+            self.assertTrue(len(f._check_similar(filter_condition4)) == 3)
+            self.assertTrue(len(f._check_similar(filter_condition5)) == 1)
+            self.assertTrue(len(f._check_similar(filter_condition6)) == 1)
```

Comment:
Perhaps it's a bit more natural to write these as `self.assertEqual(len(...), 123)`, but nothing significant really.